packages/starlight/components/SidebarSublist.astro

```diff
@@ -57,6 +57,11 @@ interface Props {
 		margin-inline-start: var(--sl-sidebar-item-padding-inline);
 		border-inline-start: 1px solid var(--sl-color-hairline-light);
 		padding-inline-start: var(--sl-sidebar-item-padding-inline);
+		overflow-wrap: break-word;
```

Comment:
This selector seems to be a bit too specific and does not cover top-level links:

<img width="323" alt="SCR-20230926-jczb" src="https://github.com/withastro/starlight/assets/494699/048c8b45-3eee-454a-be84-7914649ee36e">

To test this use-case, you can locally add a new entry to the sidebar configuration:, e.g.:

```ts
sidebar: [
  {
    label: "ThisIsAVeryLongLabelToTestTheSidebarIssues",
    link: "test",
  },
  // Existing sidebar configuration
];
```

Reply:
Ohhh good catch! I've added this use case in the commit so that we can test it.
So I've dropped this rule and added instead the following which is less specific:
```css
.top-level > li {
  overflow-wrap: anywhere;
}
```